compression_test.go

```diff
@@ -1,4 +1,4 @@
-package kafka_test
+package kafka
```

Comment:
What was the reason this package needed to be changed?

Reply:
I`m sorry,  I had an error in local imports.